maup/indexed_geometries.py

```diff
@@ -28,7 +28,17 @@ def intersections(self, geometry):
     def covered_by(self, container):
         relevant_geometries = self.query(container)
         prepared_container = prep(container)
-        return relevant_geometries[relevant_geometries.apply(prepared_container.covers)]
+
+        # If `relevant_geometries` is empty, the try will fail
+        #     with an AttributeError.  In most cases this does not
+        #     occur, so it is better to try first than to test the length.
+        try:
+            return relevant_geometries[relevant_geometries.apply(prepared_container.covers)]
+        except AttributeError:
+            if len(relevant_geometries) == 0:
+                return pandas.Series([])
```

Comment:
I'd suggest returning an empty geopandas `GeoSeries` object (that way the return type of this function is more consistent). Ultimately, since `GeoSeries` inherits from Pandas's `Series`, this is just a slight nitpick.